### PR TITLE
few new options + tiny cleanup

### PR DESCRIPTION
**Pull request by asbyth**
disabled disable enchant glint for now
added options to disable lightning bolts & armorstands (should help with framerate)
cleaned up some things
stopped keycode init from opening if browser libs arent installed

### CODE REVIEW

**Comment by RDIL** [comment]
It looks good IMO

**Comment by asbyth** [comment]
ok

File: src/main/java/cc/hyperium/config/Settings.java [hunk 1]
```diff
@@ -605,6 +605,18 @@
     @SliderSetting(name = "gui.settings.fpslimiteramount", isInt = true, min = 5, max = 60, category = IMPROVEMENTS)
     public static int FPS_LIMITER_AMOUNT = 30;
 
+    @ConfigOpt
+    @ToggleSetting(name = "gui.settings.disablelightning", category = IMPROVEMENTS)
+    public static boolean DISABLE_LIGHTNING = false;
+
+    @ConfigOpt
+    //@ToggleSetting(name = "gui.settings.keeplightinguhc", category = IMPROVEMENTS) unable to figure out how to check the users game they're playing, will add back later
+    public static boolean UHC_LIGHTNING = false;
```
Comment (CoalCoding) [inline src/main/java/cc/hyperium/config/Settings.java]: Try MinigameListener ->  getCurrentMinigameName()

Comment (CoalCoding) [reply src/main/java/cc/hyperium/config/Settings.java]: I expect for UHC you're checking for "UHC CHAMPIONS" since it reads from the scoreboard.

Comment (RDIL) [reply src/main/java/cc/hyperium/config/Settings.java]: Or whatever method the discord rp uses

Comment (RDIL) [reply src/main/java/cc/hyperium/config/Settings.java]: k thats resolved

**Pull request closed without merging**